### PR TITLE
updated sass-lint and fixed most problems

### PR DESCRIPTION
**Pull request by yodasw16**
Still needed:

- [x] Fix regex to account for responsive suffixes
- [x] After Scales updates, need to update HorizontalList mixin in pagination so it uses namespaces

### CODE REVIEW

**Comment by yodasw16** [comment]
String interpolation doesn't work on mixins so using namespaces ins't possible

**Comment by yodasw16** [comment]
Before we merge this can one of you pull down this branch and make sure the thing runs? I had a weird conflict and I'm not sure I didn't break things. After this is merged I will create a new release. @afebbraro @RCopeland

File: packages/spark-extras/components/description-table/_description-table.scss [hunk 1]
```diff
@@ -1,48 +1,55 @@
-// Variables
+// ==================================================================
+// Date Picker
```
Comment (RCopeland) [inline packages/spark-extras/components/description-table/_description-table.scss]: Should be Description Table

File: packages/spark-extras/components/highlight-board/_highlight-board.scss [hunk 2]
```diff
@@ -1,4 +1,10 @@
-// Variables
+// ==================================================================
+// Navigation
```
Comment (RCopeland) [inline packages/spark-extras/components/highlight-board/_highlight-board.scss]: Should be Highlight Board

**Pull request merged**